deployment-aws/terraform/ecs.tf

```diff
@@ -0,0 +1,112 @@
+# This allows the ECS instance to assume a particular role.
+# This enables us, for instance, to give 
+# permissions to access the S3 buckets. Note that this block alone
+# does not give that permission. The task role is what is given
+# to the ECS instances spun up.
+resource "aws_iam_role" "ecs_task_role" {
+  name = "${local.common_tags.Name}-ecs-task-role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action    = "sts:AssumeRole"
+        Effect    = "Allow"
+        Sid       = ""
+        Principal = {
+          Service = "ecs-tasks.amazonaws.com"
+        }
+      }
+    ]
+  })
+}
+
+resource "aws_iam_role_policy" "ecs_s3_access" {
+  name   = "AllowECSAccessToStorageBucket"
```

Comment:
Should this name have a `${local.common_tags.Name}` prefix for easier identification?

Reply:
yes, done